### ISSUE TITLE
Improve -Rxmin/xmax/ymin/ymax[+uunit] docs

### ISSUE THREAD

**Issue by Esteban82** [GenericMappingTools/gmt#5686]
I think that this way of indicating the region should be separated of -Rxmin/xmax/ymin/ymax. It is important to indicate that when this method is used, a longitude and latitude must be indicated in -J (important for mercator at least), as far as I know. Are those distances measured from the given long, lat to the sides? Because it could vary based on the projection (like Mercartor).

**Comment by PaulWessel** [comment]
Yes, I agree. In the current [docs](https://docs.generic-mapping-tools.org/dev/gmt.html#r-full) the w/e/s/n and this one is lumped together but as you point out the second is relative to the projection center.  I am assigning this to @meghanrjones to split that first point into two items.

**Comment by meghanrjones** [comment]
OK, will take a look and propose a suggestion for @Esteban82 to review on Monday.
